src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetUptime.cs

```diff
@@ -0,0 +1,82 @@
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights reserved.
+--********************************************************************/
+
+using System;
+using System.Management.Automation;
+using System.Diagnostics;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// This class implements Get-Uptime.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "Uptime", DefaultParameterSetName = "Timespan", HelpUri = "")]
+    public class GetUptimeCommand : PSCmdlet
+    {
+        /// <summary>
+        /// Timespan parameter
+        /// Time since the system started up
+        /// </summary>
+        /// <value></value>
+        [Parameter(ParameterSetName = "Timespan")]
+        public SwitchParameter Timespan { get; set; } = new SwitchParameter();
+
+        /// <summary>
+        /// Since parameter
+        /// The system startup time
+        /// </summary>
+        /// <value></value>
+        [Parameter(ParameterSetName = "Since")]
+        public SwitchParameter Since { get; set; } = new SwitchParameter();
+
+        /// <summary>
+        /// Pretty parameter
+        /// Convert output to default formated string
+        /// </summary>
+        /// <value></value>
+        [Parameter(ParameterSetName = "Timespan")]
+        [Parameter(ParameterSetName = "Since")]
+        public SwitchParameter Pretty { get; set; } = new SwitchParameter();
+
+        /// <summary>
+        /// ProcessRecord() override.
+        /// This is the main entry point for the cmdlet.
+        /// </summary>
+        protected override void ProcessRecord()
+        {
+            // Get-Uptime return null if IsHighResolution = false  
+            // because stopwatch.GetTimestamp() return DateTime.UtcNow.Ticks
+            // instead of ticks from system startup
```

Comment:
I take back my previous comment. Silently returning `null` is bad in the following example:
```
$timespan = [timespan]::new(24, 0, 0) # time span of 24 hours 
if ((Get-Uptime) -gt $timespan) {
    ## take the server offline for maintenance
}
```
In this case, the server may never get maintained 😦